src/modules/shared/pages/innovation/support/support-summary-list.component.html

```diff
@@ -59,19 +59,20 @@ <h2 [ngClass]="{ 'nhsuk-u-padding-top-3': !sectionIsFirst }">{{ sectionItem.titl
                           <span class="nhsuk-u-font-weight-bold">Updated support status</span>
                           <span class="nhsuk-hint nhsuk-u-font-size-14">
                             {{ historyItem.createdAt | date: ('app.date_formats.medium_date' | translate) }}
-                            by {{ historyItem.createdBy.name }}{{ historyItem.createdBy.displayRole ? ' (' + historyItem.createdBy.displayRole + ')' : '' }}
+                            by <strong>{{ historyItem.createdBy.name }}{{ historyItem.createdBy.displayRole ? ' (' + historyItem.createdBy.displayRole + ')' : '' }}</strong>
                           </span>
                         </p>
                         <theme-tag type="{{ 'shared.catalog.innovation.support_status.' + historyItem.params.supportStatus + '.cssColorClass' | translate }}" label="{{ 'shared.catalog.innovation.support_status.' + historyItem.params.supportStatus + '.name' | translate }}"></theme-tag>
-                        <p *ngIf="historyItem.params.message" class="nhsuk-body-s nhsuk-u-margin-top-2">{{ historyItem.params.message }}</p>
+                        <p class="nhsuk-u-font-weight-bold font-color-secondary nhsuk-u-margin-top-2 nhsuk-u-margin-bottom-2">Message</p>
+                        <p *ngIf="historyItem.params.message" class="nhsuk-body-s nhsuk-u-margin-bottom-2">{{ historyItem.params.message }}</p>
```

Comment:
It is better to keep the `*ngIf` outside both paragraphs. Imagine if you don't have a message, it will just be a hanging title there.